db/fixtures/miq_product_features.yml

```diff
@@ -5270,6 +5270,10 @@
           :description: Remove a Repository
           :feature_type: admin
           :identifier: embedded_configuration_script_source_delete
+        - :name: Refresh Repositories
+          :description: Ansible Repositories Refresh
+          :feature_type: control
+          :identifier: embedded_automation_manager_repository_refresh
```

Comment:
@mzazrivec should this one be embedded_configuration_script_source_refresh to be consistent